### ISSUE TITLE
ajax mode ?

### ISSUE THREAD

**Issue by paskuale75** [fedemotta/yii2-widget-datatables#25]
this widget work only with DataProvider models ? Any suggestion for work with serverSide option ? Thanks

**Comment by YevhenIvanovych** [comment]
Hi.
And I support the question.

**Comment by ghost** [comment]
Yes is there any way to apply the script on my HTML DOM table using this extension ?
